rice/detail/NativeCallbackFFI.ipp

```diff
@@ -117,8 +117,8 @@ namespace Rice::detail
     }
 
     // Create FFI closure
-    this->closure_ = (ffi_closure *)ffi_closure_alloc(sizeof(ffi_closure), (void**)(&this->callback_));
-    ffi_status status = ffi_prep_closure_loc(this->closure_, &cif_, ffiCallback, (void*)this, nullptr);
+    this->closure_ = (ffi_closure *)ffi_closure_alloc(sizeof(ffi_closure) + sizeof(void*), (void**)(&this->callback_));
```

Comment:
Resizing this to be slightly bigger than `ffi_closure` may not be necessary but I saw this a lot in other code bases using this same API.

Reply:
Cool - thanks for figuring this out. And replacing the nullptr parameter.